packages/cra-template-edu/template/src/pages/Root.tsx

```diff
@@ -0,0 +1,24 @@
+import {Heading} from '@chanzuckerberg/eds';
+import {useQuery} from '@tanstack/react-query';
+import {getTemp} from '../api/weather';
+
+const sfCoord = {lat: 37.77, long: -122.43};
+
+export default function Root() {
+  const query = useQuery(
+    // Query key, which should be unique to this data. Include any query params or ids.
+    // https://tanstack.com/query/v4/docs/guides/query-keys
+    ['temp', sfCoord.lat, sfCoord.long],
+    // Function that actually makes the HTTP request.
+    // https://tanstack.com/query/v4/docs/guides/query-functions
+    () => getTemp(sfCoord.lat, sfCoord.long),
+  );
+
+  return (
+    <main className="p-5">
+      <Heading size="h2">Hello world</Heading>
+      <p>Lorem ipsum dolor sit amet, consectetur adip.</p>
+      <p>Current San Francisco temp: {query.data ?? '🤔'}</p>
```

Comment:
Todo: extract to an SFWeather component, and add msw for testing.

Reply:
nvm. Adding MSW as part of the template is hard because [create-react-app removes any "browser" field we try to add to the package.json](https://github.com/facebook/create-react-app/blob/9358f50bae992ef1083b9590ca1a9874c4c254cd/packages/react-scripts/scripts/init.js#L152). Unfortunately, we need to do that to prevent Storybook from trying to load MSW's Node integration :/